From e30e0b8fc689f25d4e211c46c3aa80a4b00b2084 Mon Sep 17 00:00:00 2001
From: Christian Dupuis <cd@docker.com>
Date: Wed, 28 May 2025 23:14:34 +0200
Subject: [PATCH] Update org.eclipse.jgit to 7.2.1.202505142326-r
Reference: https://projects.eclipse.org/projects/technology.jgit/releases/7.2.1/
Description: Rebase the existing JGit override onto Gradle 8.14.6, which
 pins JGit 5.13.5.202508271544-r.

---
 build-logic-commons/build-platform/build.gradle.kts   | 2 +-
 packaging/distributions-dependencies/build.gradle.kts | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/build-logic-commons/build-platform/build.gradle.kts b/build-logic-commons/build-platform/build.gradle.kts
index 8eca912a8be..764b5d10c41 100644
--- a/build-logic-commons/build-platform/build.gradle.kts
+++ b/build-logic-commons/build-platform/build.gradle.kts
@@ -78,7 +78,7 @@ dependencies {
         api("org.codehaus.groovy:groovy:$groovyVersion")
         api("org.codehaus.groovy.modules.http-builder:http-builder:0.7.2") // TODO maybe change group name when upgrading to Groovy 4
         api("org.codenarc:CodeNarc:$codenarcVersion")
-        api("org.eclipse.jgit:org.eclipse.jgit:5.13.5.202508271544-r")
+        api("org.eclipse.jgit:org.eclipse.jgit:7.2.1.202505142326-r")
         api("org.javassist:javassist:3.30.2-GA")
         api("org.jetbrains.kotlinx:kotlinx-metadata-jvm:0.9.0")
         api("org.jsoup:jsoup:1.23.2")
diff --git a/packaging/distributions-dependencies/build.gradle.kts b/packaging/distributions-dependencies/build.gradle.kts
index aa8d03e3d75..7c2880b76ae 100644
--- a/packaging/distributions-dependencies/build.gradle.kts
+++ b/packaging/distributions-dependencies/build.gradle.kts
@@ -122,8 +122,8 @@ dependencies {
         api(libs.jclToSlf4j)            { version { strictly(slf4jVersion) }}
         api(libs.jcommander)            { version { strictly("1.78") }}
         api(libs.jetbrainsAnnotations)  { version { strictly("24.0.1") }}
-        api(libs.jgit)                  { version { strictly("5.13.5.202508271544-r"); because("6.x requires Java 11") }}
-        api(libs.jgitSsh)               { version { strictly("5.13.5.202508271544-r") }}
+        api(libs.jgit)                  { version { strictly("7.2.1.202505142326-r"); because("6.x requires Java 11") }}
+        api(libs.jgitSsh)               { version { strictly("7.2.1.202505142326-r") }}
         api(libs.joda)                  { version { strictly("2.12.2") }}
         api(libs.joptSimple)            { version { strictly("5.0.4"); because("needed to create profiler in Gradle profiler API") }}
         api(libs.jsch)                  { version { strictly("0.2.16") }}
-- 
2.49.0
